lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/aplinear/APLExplanationResult.java

```diff
@@ -30,41 +27,58 @@ public APLExplanationResult(
         this.metrics = metrics;
     }
 
-    @Override
-    public String toString() {
-        return "a="+Arrays.toString(matcher.values)+":ag="+Arrays.toString(aggregates)+":mt="+Arrays.toString(metrics);
-    }
-
-    public String prettyPrint(
-            AttributeEncoder encoder,
-            List<String> aggregateNames
-    ) {
+    private Map<String, String> prettyPrintMatch(AttributeEncoder encoder) {
         Set<Integer> values = matcher.getSet();
+        Map<String, String> match = new HashMap<>();
 
-        StringJoiner matchJoiner = new StringJoiner(", ");
         for (int k : values) {
-            matchJoiner.add(encoder.decodeColumnName(k)+"="+encoder.decodeValue(k));
+            match.put(encoder.decodeColumnName(k), encoder.decodeValue(k));
         }
+        return match;
+    }
+
+    private Map<String, String> prettyPrintMetric() {
```

Comment:
let's rename these to metricToMap, matchToMap, etc... since they don't actually print they just convert to maps